index.html

```diff
@@ -203,9 +383,55 @@ <h2 class="work-title">tonic</h2>
           <li>css</li>
           <li>Javascript</li>
         </ul>
-        <div>
-          <button type="button" class="btn">See Project</button>
-        </div>
+        <button data-modal-target="#modal1" class="btn">See Project</button>
+
+          <div class="modal1" id="modal1">
+            <div class="modal1-description">
+              <h2 class="work-title">tonic</h2>
+              <button data-btn-close class="btn-close1">&times;</button>
+            </div>
+
+            <div class="modal1-body">
+              <ul class="canopy">
+                <li>canopy</li>
+                <li>Back End Dev</li>
+                <li>2015</li>
+              </ul>
+              <div class="work-img">
+                <img src="images/Snapshoot-Portfolio1.svg" alt="tonic image" />
+              </div>
+              <p class="work-modal1-details">
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it to make a type specimen book. It has
+                survived not only five centuries, but also the leap into
+                electronic typesetting, remaining essent
+              </p>
+              <br>
+              <ul class="work-tools">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+              </ul>
+              <br />
+              <div>
+                <button type="button" class="btn">
+                  <span class="btn-text">See live</span>
+                  <span class="btn-icon">
+                    <img src="images/live-icon.svg" alt="Icon" />
+                  </span>
+                </button>
+                <button type="button" class="btn">
+                  <span class="btn-text">See source</span>
+                  <span class="btn-icon">
+                    <img src="images/github-icon.svg" alt="github-Icon" />
+                  </span>
+                </button>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] [OPTIONAL] I'm making this optional because it may have been omitted in the previous reviews. It is also a recommendation that the Modal popup data is populated with the data from the projects array